src/content/structured/components/badge/code.mdx

```diff
@@ -189,12 +192,12 @@ export const snippetsInformation = [
   {
     language: "Web component",
     snippet: `<ic-button variant="secondary" style="margin-right:16px">
-  <ic-badge slot="badge" variant="info"></ic-badge>
-  Dot
+  <ic-badge slot="badge" type="text" text-label="8" variant="info"></ic-badge>
```

Comment:
This section needs re-titling. 'With information' doesn't make much sense. Rename to 'Badge types'